tests/python_package_test/test_engine.py

```diff
@@ -1258,14 +1263,13 @@ def test_cv():
     np.testing.assert_allclose(cv_res_lambda["valid ndcg@3-mean"], cv_res_lambda_obj["valid ndcg@3-mean"])
 
 
-def test_cv_works_with_init_model(tmp_path):
+def test_cv_works_with_init_model(model_path_txt):
     X, y = make_synthetic_regression()
     params = {"objective": "regression", "verbose": -1}
     num_train_rounds = 2
     lgb_train = lgb.Dataset(X, y, free_raw_data=False)
     bst = lgb.train(params=params, train_set=lgb_train, num_boost_round=num_train_rounds)
     preds_raw = bst.predict(X, raw_score=True)
-    model_path_txt = str(tmp_path / "lgb.model")
```

Comment:
Since this was already using `tmp_path`, I think it's very unlikely that this one was leaving a file behind. Could you please revert the changes to this case and any others that were already using `tmp_path`?